### ISSUE TITLE
Macbook pro 2019 i9 no stable boot

### ISSUE THREAD

**Issue by Aisub7** [rescuezilla/rescuezilla#355]
Hi, 

One time I managed to successfully clone and restore an image. Clonezilla 2.4 now only boots till the different startup modes (startup, use graphical fallback mode, etc,) The display, keyboard, and mouse drivers do not appear to function. You have to use external peripherals. I'll try to document the code that appears on the screen before boot.

**Comment by shasheene** [comment]
Yep, I screwed up the video drivers on Rescuezilla v2.4 leading to blank screens on most graphics hardware after I updated to Ubuntu 22.04 "Jammy". Based on your report I may have also screwed up keyboard and mouse drivers? That seems super strange and not in line with other reports, but I'll investigate.

I was hoping the fact my new "weekly rolling releases" model (where what became Rescuezilla v2.4 was released publicly 2 weeks ago) would have helped catch issues like this before the stable release, but it unfortunately didn't end up helping here :disappointed: 

**Expect an updated Rescuezilla v2.4.1 release hopefully this weekend.**

Using "Graphical Fallback Mode" may help, or you can stick to Rescuezilla v2.3.1 (which has two variants available: "Focal" and "Impish") available on the Releases page under the "Assets" section.

**Comment by Aisub7** [comment]
I have to apologise, looks like the USB was the culprint. From my last boot camp windows experience I remember the trackpad also not functioning before injecting the apple drivers. 

Using graphical fallback results in a good high resolution screen above 1920x1080.

Thank you for the fast response.
